src/main/java/com/enonic/app/rewrite/PathBuilder.java

```diff
@@ -0,0 +1,36 @@
+package com.enonic.app.rewrite;
+
+public class PathBuilder
+{
+    private final String context;
+
+    private final String path;
+
+    private PathBuilder( final String context, final String path )
+    {
+        this.context = normalize( context );
+        this.path = normalize( path );
+    }
+
+    public static PathBuilder create( final String context, final String path )
+    {
+        return new PathBuilder( context, path );
+    }
+
+    private String normalize( String value )
+    {
+        if ( value != null )
+        {
+            return value.startsWith( "/" ) ? value.replaceFirst( "/", "" ) : value;
+        }
+        else
+        {
+            return "";
+        }
+    }
+
+    public String build()
+    {
+        return context + "/" + path;
```

Comment:
After all this "builder" just concatenates two strings with "/", it can be a singe function.

Reply:
it was discussed